src/Login.re

```diff
@@ -0,0 +1,9 @@
+open Chrome.Extensions;
```

Comment:
I suggest removing the `open Chrome.*` statements in all the `*.re` files for now.
They are unused, we can add them whenever we actually need them